pep-0701.rst

```diff
@@ -320,7 +336,75 @@ All restrictions mentioned in the PEP are lifted from f-string literals, as expl
   expanded when the innermost string is evaluated.
 * Comments, using the ``#`` character, are possible only in multi-line f-string literals,
   since comments are terminated by the end of the line (which makes closing a
-  single-line f-string literal impossible)
+  single-line f-string literal impossible). Comments in multi-line f-string literals require
+  the closing ``{`` of the expression part to be present in a different line as the one the
+  comment is in.
+
+.. _considerations_of_quote_reuse:
+
+Considerations regarding quote reuse
+------------------------------------
+
+One of the consequences of the grammar proposed here is that, as mentioned above,
+f-string expressions can now contain strings delimited with the same kind of quote
+that is used to delimit the external f-string literal. For example:
+
+    >>> f" something { my_dict["key"] } something else "
+
+In the `discussion thread of this document <https://discuss.python.org/t/pep-701-syntactic-formalization-of-f-strings>`_
+several concerns have been raised regarding this aspect and we want to collect them here
+as these should be taken into consideration when accepting or rejecting this document.
+
+Some of these objections include:
+
+* Many people find quote reuse withing the same string confusing and hard to read. This is because
+  allowing quote reuse will violate a current property of Python as it stands today: the fact that
+  strings are fully delimited by the same kind of quote, which by itself is a very simple rule. It
+  has been pointed out that one of the reasons quote reuse may be harder for humans to parse and
+  will lead to code that may be harder to read is that the quote character is the same on start and
+  end (as opposed to other delimiters). 
```

Comment:
This sentence is quite long and hard to parse; here's a suggested rewording. I'm not quite sure what's meant by "same on start and end" here; you could consider rephrasing that as well.

```suggestion
  strings are fully delimited by the same kind of quote, which by itself is a very simple rule.
  One of the reasons quote reuse may be harder for humans to parse, leading to less readable
  code, is that the quote character is the same on start and
  end (as opposed to other delimiters). 
```

Reply:
> what's meant by "same on start and end"

That the delimiter is the same at the beginning at the end (as opposed to parentheses where "(" is different than ")".).